projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-item.php

```diff
@@ -122,6 +163,20 @@ public function large_file() {
  * Tiled gallery rectangular item class.
  */
 class Jetpack_Tiled_Gallery_Rectangular_Item extends Jetpack_Tiled_Gallery_Item { // phpcs:ignore Generic.Files.OneObjectStructurePerFile.MultipleFound, Generic.Classes.OpeningBraceSameLine.ContentAfterBrace
+	/**
+	 * The image URL, in grayscale.
+	 *
+	 * @var string
+	 */
+	public $img_src_grayscale;
+
+	/**
+	 * The image size.
+	 *
+	 * @var string
+	 */
+	public $size;
```

Comment:
Similarly, the way the property is used it probably belongs on the parent class  as `string|null`.

https://github.com/Automattic/jetpack/blob/9a3a10ad9562ee5649d39ad815a52a49fb258783/projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/templates/partials/item.php#L18-L19